src/index.js

```diff
@@ -87,7 +86,20 @@ var mq = React.createClass({
       return null;
     }
     var props = omit(this.props, excludedPropKeys);
-    return React.createElement(this.props.component, props, this.props.children);
+    if (this.props.component || this.props.children.length > 1) {
```

Comment:
FYI `children` is an opaque data structure and its internal representation can change. It’s better to use `React.Children.count(this.props.children)`.